flink-runtime-web/src/test/resources/rest_api_v1.snapshot

```diff
@@ -4190,4 +4190,4 @@
       }
     }
   } ]
-}
```

Comment:
Ah, it's better to revert this change

Reply:
This change was reverted, and the commits merge and rebase have been completed.